public/admin/index.html

```
@@ -1191,6 +1203,122 @@ <h4>Broadcast</h4>
+                </label>
+                <label class="form-inline">
+                  <span>Replay policy</span>
+                  <select id="challengeAdminReplayPolicy">
+                    <option value="best">best</option>
+                    <option value="first-only">first-only</option>
+                    <option value="unlimited">unlimited</option>
+                  </select>
+                </label>
+                <label class="form-inline">
+                  <span>Start time (optional)</span>
+                  <input type="datetime-local" id="challengeAdminStartTime" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)